packages/cms-lib/lib/config.js

```diff
@@ -353,6 +377,118 @@ const deleteEmptyConfigFile = () => {
   );
 };
 
+const getConfigVariablesFromEnv = () => {
+  const env = process.env;
+
+  return {
+    apiKey: env[HUBSPOT_API_KEY],
+    clientId: env[HUBSPOT_CLIENT_ID],
+    clientSecret: env[HUBSPOT_CLIENT_SECRET],
+    personalAccessKey: env[HUBSPOT_PERSONAL_ACCESS_KEY],
+    portalId: parseInt(env[HUBSPOT_PORTAL_ID], 10),
+    refreshToken: env[HUBSPOT_REFRESH_TOKEN],
+  };
+};
+
+const generatePersonalAccessKeyPortalConfig = (portalId, personalAccessKey) => {
+  return {
+    defaultPortal: ENVIRONMENT_VARIABLES_DEFAULT_PORTAL_NAME,
+    portals: [
+      {
+        inMemoryOnly: true,
+        name: ENVIRONMENT_VARIABLES_DEFAULT_PORTAL_NAME,
+        authType: PERSONAL_ACCESS_KEY_AUTH_METHOD.value,
+        portalId,
+        personalAccessKey,
+      },
+    ],
+  };
+};
+
+const generateOauthPortalConfig = (
+  portalId,
+  clientId,
+  clientSecret,
+  refreshToken,
+  scopes
+) => {
+  return {
+    defaultPortal: ENVIRONMENT_VARIABLES_DEFAULT_PORTAL_NAME,
+    portals: [
+      {
+        inMemoryOnly: true,
+        name: ENVIRONMENT_VARIABLES_DEFAULT_PORTAL_NAME,
+        authType: OAUTH_AUTH_METHOD.value,
+        portalId,
+        auth: {
+          clientId,
+          clientSecret,
+          scopes,
+          tokenInfo: {
+            refreshToken,
+          },
+        },
+      },
+    ],
+  };
+};
+
+const generateApiKeyPortalConfig = (portalId, apiKey) => {
+  return {
+    defaultPortal: ENVIRONMENT_VARIABLES_DEFAULT_PORTAL_NAME,
```

Comment:
What is the thinking behind incorporating a default portal into the environment variables system.

Reply:
I was using this so the in-memory portal config that is generated using the environment variables would be used by default

Reply:
What if we modify https://github.com/HubSpot/hubspot-cms-tools/blob/0074dc1a177d4335fd2e838c75a20d51a76acb47/packages/cms-cli/lib/commonOpts.js#L46 instead? We probably also want to throw an error if `--portal` is passed as an option.